frontend/libs/studio-components/src/components/StudioExpression/SimplifiedEditor/LogicalExpressionEditor/SubExpression/SubExpressionValueSelector/SubExpressionValueContentInput/ComponentIdSelector.module.css

```
@@ -0,0 +1,4 @@
+.comboboxBackground {
```

Suggest to simplify the class name, since we do not need to inform about implementation details of the CSS. 😊 
```suggestion
.combobox {
```

---

I agree, but on the other hand, if this class's only role is to style the background, doesn't that count for something?😄

---

This may be a personal preference, but when I'm reading the JSX I'm not interested in the implementation details of the CSS class itself. I see a class, where I can go into the class to see what it does if needed. Another reason to avoid implementation details in the class names is that when someone adds something more they do not need to update the class name. Even worse, what if they forgot to update the class name after adding more properties later on?

But again, this may be a personal preference. 😊

---

Yeah that is a good argument that it is not flexible with later changes, and I agree. Was just curious about your thoughts. :)
Will change the name if I keep the styling, but will probably remove it when I have investigated the background bug in the first comment.